modules/sticky-table-headers/main/index.coffee

```diff
@@ -258,28 +274,33 @@ class StickyTableHeaders
 
       topLeftTable = topLeftHead.append tableClone.clone(true)
 
-      realNodes = table.select('thead').selectAll('tr').select('th, td').nodes
-      clonedNodes = topLeftTable.select('thead').selectAll('tr').select('th, td').nodes
-
-      for i in [0...realNodes.length]
-        cloneEvents(realNodes[i], clonedNodes[i])
-        hx.select(realNodes[i]).classed('hx-sticky-table-invisible', true)
+      realNodes = getChildrenFromTable table, false, true
+      clonedNodes = getChildrenFromTable topLeftTable, false, true
 
+      createStickyHeaderNodes realNodes, clonedNodes
 
     topHead?.style('height', offsetHeight + 'px')
       .style('width', wrapperBox.width + 'px')
       .style('left', offsetWidth + 'px')
+      .selectAll('.hx-sticky-table-invisible')
```

Comment:
The `hx-sticky-table-invisible` class is added to the headers (left, top, top-left) in the main table so they are invisible but still affect the layout (`visibility: hidden`). 
The class isn't needed inside the cloned tables and is removed here as otherwise it causes problems with nested tables due to the styles:
```
.hx-sticky-table-wrapper .hx-sticky-table-invisible {
    visibility: hidden;
}  
```

It's a lot easier to remove the class than add nesting to the css.

Also, the class is added in `createStickyHeaderNodes` (line 60) after the clone has been made, however the class applied to the original is inherited by the clone and removing the class from the clone didn't work, hence why this removal is required.